.github/workflows/build.yml

```diff
@@ -3,6 +3,14 @@ on:
   push:
     branches:
       - master
+env:
+  # Variables defined in the repository
+  SENTRY_ORG: ${{ vars.SENTRY_ORG }}
+  # For master we have an environment variable that selects the action-release project
+  # instead of action-release-prs
+  # For other branches: https://sentry-ecosystem.sentry.io/releases/?project=4505075304693760
+  # For master branch: https://sentry-ecosystem.sentry.io/releases/?project=6576594
+  SENTRY_PROJECT: ${{ vars.SENTRY_PROJECT }}
```

Comment:
You can see in the following screenshot how I pick the right project name based on the branch used:
<img width="706" alt="image" src="https://user-images.githubusercontent.com/44410/234890917-d629caf1-8385-498e-b2cf-8d0c2123173d.png">